src/control/server/ctl_smd_rpc.go

```diff
@@ -340,6 +349,33 @@ func addManageRespIDOnFail(log logging.Logger, res *ctlpb.SmdManageResp_Result,
 	}
 }
 
+// Retry dev-replace requests as state propagation may take some time after set-faulty call has
+// been made to manually trigger a faulty device state.
+func replaceDevRetryBusy(ctx context.Context, log logging.Logger, e Engine, req proto.Message) (res *ctlpb.SmdManageResp_Result, err error) {
+	for try := uint(0); try < uint(maxDevReplaceRetries); try++ {
```

Comment:
nit - any reason to cast these to uints? Or could they be plain ints?

Reply:
I think we discussed this on the Master PR-12779, these could be changed back to int and I will do that in the future PR, would like to keep this backport clean if possible

Reply:
fix for this applied on https://github.com/daos-stack/daos/pull/12944